src/quo/components/tags/summary_tag/style.cljs

```diff
@@ -4,7 +4,7 @@
 
 (defn main
   [{:keys [type customization-color]} theme]
-  {:justify-content  :center
+  {:justify-content  :flex-start
```

Comment:
Have you confirmed that other usage of tags are not affected ?

Reply:
I checked the preview, but that's about it

Reply:
it's there already